src/sentry/static/sentry/app/components/events/contexts/app/app.tsx

```diff
@@ -19,8 +20,15 @@ const appKnownDataValues = [
   AppKnownDataType.BUILD,
 ];
 
+const appIgnoredDataValues = [];
```

Comment:
I see only `contexts.os.build` is ignored by default. Perhaps we can leave out all the stubs for other contexts? YAGNI.

Reply:
I believe `contexts.runtime.build` is also ignored. YAGNI might be true, however, all of these context types follow a predefined structure so that they are all unified and in a predictable form. I would like to keep it that way.

I also suspect that after merging this there might be values popping up we did not even know about from various SDKs. This way other devs can quickly get rid of them without reinventing the logic.